AdminWebsite/AdminWebsite.UnitTests/Controllers/JusticeUserController/EditJusticeUserTests.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using AdminWebsite.Controllers;
+using AdminWebsite.Models;
+using AdminWebsite.Testing.Common.Builders;
+using Autofac.Extras.Moq;
+using BookingsApi.Client;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Contract.Responses;
+using FizzWare.NBuilder;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+
+namespace AdminWebsite.UnitTests.Controllers.JusticeUserController
+{
+    public class EditJusticeUserTests
+    {
+        private JusticeUsersController _sut;
+        private AutoMock _mocker;
+        private readonly string _username = "test.vho@hmcts.net";
+
+        [SetUp]
+        public void Setup()
+        {
+            _mocker = AutoMock.GetLoose();
+            _sut = SetupControllerWithClaims();
+        }       
+
+        private JusticeUsersController SetupControllerWithClaims()
+        {
+            var cp = new ClaimsPrincipalBuilder().WithRole(AppRoles.VhOfficerRole)
+                .WithUsername(_username).Build();
+            var context = new ControllerContext
```

Comment:
I don't think we need to set the claims here, as the user identity isn't being referenced in the controller action.

We should be able to simplify this to:

```
private JusticeUsersControlloer SetupController()
{
  var controller = _mocker.Create<JusticeUsersController>();
  return controller;
}
```

Reply:
Ah, nice - thanks Oliver. Done.

Reply:
great. A couple of other things to follow on from that:

- Maybe rename the method to `SetupController`, as it no longer sets up claims.
- `private readonly string _username = "test.vho@hmcts.net";` (line 25 of the same file) - this is no longer used, so we should be able to remove that.

Reply:
Got it - done. Thanks!